purview/utils/general.py

```diff
@@ -183,6 +185,12 @@ def query_neo4j(query):
     if not results:
         return results_dict
 
+    if isinstance(results[0][0], int):
+        return results[0][0]-1
```

Comment:
When creating the separate function, don't forget to remove `-1` and have that done where the function is called so the separate function is generic.